tap_jira/streams.py

```diff
@@ -4279,14 +3570,21 @@ class IssueChangeLogStream(JiraStream):
 
     path = "/issue/{issue_id}/changelog"
 
+    replication_method = "INCREMENTAL"
+
```

Comment:
This is redundant since you're declaring `replication_key`

```suggestion
```

Reply:
okay, thanks for this. Will be adding a new stream and then push this too.